### PR TITLE
Added form and location list

### PR DESCRIPTION
**Pull request by borjanpopovic**
Added form.
Entered details and chose map pin are rendered to the location list (please ignore currentPosition option yet)
Styling still provisory

### CODE REVIEW

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -31,8 +32,9 @@ export default createGlobalStyle`
    *::before,
    *::after {
      box-sizing: border-box;
-     background: var(--primary);
+     //ackground: var(--primary);
```
Comment (SabMueller) [inline src/GlobalStyle.js:35]: do you actually need the background property? it seems like you don't, so you can delete the comment as well :)

File: src/pages/Locationlist.js [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+
+/* const H1 = styled.h1`
+  font-size: 1.25rem;
+  margin-top: 5rem;
+  margin-bottom: 1rem;
+  color: var(--border);
+  text-align: center;
+  padding-top: 2rem;
+  z-index: 100;
+`; */
```
Comment (SabMueller) [inline src/pages/Locationlist.js:30]: I guess you could delete this comment as well? :)

File: src/pages/form.js [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
+`;
 
+/* const H1 = styled.h1`
+  font-size: 1.25rem;
+  margin-top: 5rem;
+  margin-bottom: 1rem;
+  color: var(--border);
+  text-align: center;
+  padding-top: 2rem;
+`; */
```
Comment (SabMueller) [inline src/pages/form.js:31]: what about this one? :D

**Review by SabMueller — APPROVED** [review]
Looks good to me! The only remarks i had were about some line comments. You should consider if you actually plan to use them, otherwise you might as well delete them. Keep up the good work! ✨ 🥇

File: src/GlobalStyle.js (hunk 1 above)
Comment (borjanpopovic) [reply src/GlobalStyle.js:35]: Thank you for the hint! I left some CSS tryouts as comments. I'll come back to them and delete them.

File: src/pages/Locationlist.js (hunk 2 above)
Comment (borjanpopovic) [reply src/pages/Locationlist.js:30]: Same here as above :)

File: src/pages/form.js (hunk 3 above)
Comment (borjanpopovic) [reply src/pages/form.js:31]: Same :) thanks.

**Comment by borjanpopovic** [comment]
Thank you for your review!!

**Pull request merged**